libs/providers/go-feature-flag-web/src/lib/go-feature-flag-web-provider.spec.ts

```diff
@@ -214,7 +214,7 @@ describe('GoFeatureFlagWebProvider', () => {
     it('should get a valid boolean flag evaluation', async () => {
       const flagKey = 'bool_flag';
       await OpenFeature.setContext(defaultContext);
-      OpenFeature.setProvider('test-provider', defaultProvider);
+      await OpenFeature.setProviderAndWait('test-provider', defaultProvider);
```

Comment:
@thomaspoignant @beeme1mr These tests change due to our new "short circuiting logic" (there weren't ready since we weren't awaiting the init).